Sources/Moya/URLRequest+Encoding.swift

```diff
@@ -6,6 +6,11 @@ internal extension URLRequest {
         do {
             let encodable = AnyEncodable(encodable)
             httpBody = try JSONEncoder().encode(encodable)
+
+            if value(forHTTPHeaderField: "Content-Type") == nil {
```

Comment:
Maybe we can add a local constant `let contentType = "Content-Type"` here and use that instead of typing it twice? To prevent regressions in the future.

Reply:
Sounds like a good idea to me. I pushed the change.